modules/combined/test/tests/umat/tests

```diff
@@ -8,6 +8,7 @@
     exodiff = 'gap_heat_transfer_umat_out.e'
     library_mode = 'DYNAMIC'
     valgrind = 'NONE'
+    skip_installed = True
```

Comment:
Haha - no way!
You really want to avoid these double negative combinations. I know it would be rare to only run a test in installed mode, but if we do that option would be `skip_installed = False`. 

Perhaps it should be an enum: `build_location = [IN_TREE, INSTALLED, BOTH]`?
The default would be `BOTH`.

Reply:
👍 
The way `compiler =` works is what I will follow:
```pre
set(['ALL', 'IN_TREE', 'INSTALLED'])
```